packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/app/assertion-generators/json.cjs

```diff
@@ -0,0 +1,16 @@
+// Not guaranteed to sort all array elements and weird keys properly, but definitely good enough for the usecase
+module.exports = function stableOrderJson(obj) {
+  return JSON.stringify(obj, (_, v) => {
+    if (v && v.constructor === Object) {
```

Comment:
couldn't do `v?.` because node 12. my golf score suffers

Reply:
I’ve dropped Node.js 12 from CI if that makes a difference.

Reply:
The idiom we favor here is `Array.isArray(v)` then `Object(v) === v` for the isObject test.

https://github.com/endojs/endo/blob/master/packages/ses/src/commons.js#L233-L250